include/seqan3/alphabet/concept.hpp

```diff
@@ -75,8 +119,8 @@ namespace seqan3
  *
  * It acts as a wrapper and looks for three possible implementations (in this order):
  *
- *   1. A free function `to_rank(your_type const a)` in the namespace of your type (or as `friend`).
- *   2. A free function `to_rank(your_type const a)` in `namespace seqan3::custom`.
+ *   1. A static member function `to_rank(your_type const a)` of the class `seqan3::custom<your_type>`.
```

Comment:
```suggestion
 *   1. A static member function `to_rank(your_type const a)` of the class `seqan3::custom::alphabet<your_type>`.
```
Adapt documentation everywhere else too